common/logger/Cargo.toml

```diff
@@ -13,13 +13,11 @@ edition = "2018"
 # This is to avoid ever having a circular dependency with the libra-logger crate.
 [dependencies]
 chrono = "0.4.15"
-env_logger = { version = "0.7.1", default-features = false }
 erased-serde = "0.3.12"
 hex = "0.4.2"
 libra-log-derive = { path = "derive", version = "0.1.0" }
 libra-time = { path = "../time", version = "0.1.0" }
 libra-workspace-hack = { path = "../workspace-hack", version = "0.1.0" }
-log = "0.4.11"
```

Comment:
curious if you double checked if there were other uses in the code base

./language/move-prover/spec-lang/Cargo.toml:log = "0.4.11"
./language/move-prover/Cargo.toml:log = "0.4.11"
./language/move-prover/Cargo.toml:simplelog = "0.8.0"
./language/move-prover/abigen/Cargo.toml:log = "0.4.11"
./language/move-prover/docgen/Cargo.toml:log = "0.4.11"
./language/move-prover/errmapgen/Cargo.toml:log = "0.4.11"
./language/stdlib/Cargo.toml:log = "0.4.11"
./language/compiler/ir-to-bytecode/Cargo.toml:log = "0.4.11"
./common/logger/Cargo.toml:log = "0.4.11"
./devtools/x/Cargo.toml:log = "0.4.11"


stdlib probably should be unified ... the rest optional but it does suck to have two distinct logging crates... would be helpful to add a lint if it is possible.

Reply:
Yes I was aware that we're using the log crate still in a number of other places. Thankfully none of those other crates are actually a part of the production node binary. This is still on my list of things to things to improve/fix.